package.json

```diff
@@ -10,8 +10,10 @@
   },
   "dependencies": {
     "autoprefixer": "^10.0.2",
+    "dotenv": "^8.2.0",
     "gray-matter": "^4.0.2",
     "marked": "^1.2.4",
+    "mongoose": "^5.11.13",
```

Comment:
I don't wanna use mongo or mongoose for this project.

Reply:
I'll remove it